docs/rst/user_manual/configuration.rst

```diff
@@ -8,7 +8,8 @@ DDS Router Configuration
 
 A |ddsrouter| is configured by a *.yaml* configuration file.
 This *.yaml* file contains all the information regarding the |ddsrouter| configuration, such as topics filtering
-and :term:`Participants <Participant>` configurations.
+and :term:`Participants <Participant>` configurations. Configuration files may be easily validated by using the
+:ref:`yaml_validator` tool.
```

Comment:
```suggestion
:ref:`yaml_validator` tool (see :ref:`yaml_validator`).
```

Reply:
Did you mean the following?

YAML validator tool (see :ref:`yaml_validator`).